ote/src/clj/ote/integration/import/gtfs.clj

```diff
@@ -271,25 +266,29 @@
 
 (defmethod load-transit-interface-url :gtfs [type db interface-id url last-import-date saved-etag force-download?]
   (let [response (load-interface-url db interface-id url last-import-date saved-etag force-download?)]
-    (when response
+    (if response
       (try
         (check-interface-zip type db interface-id url (java.io.ByteArrayInputStream. (:body response)))
         response
 
         ;; Return nil response in case of error
         (catch Exception e
-          nil)))))
+          nil))
+      ;; Return nil response in case of error
+      nil)))
 
 (defmethod load-transit-interface-url :kalkati [type db interface-id url last-import-date saved-etag force-download?]
   (let [response (load-interface-url db interface-id url last-import-date saved-etag force-download?)]
-    (when response
+    (if response
       (try
         (check-interface-zip type db interface-id url (java.io.ByteArrayInputStream. (:body response)))
         (update response :body kalkati-to-gtfs/convert-bytes)
 
         ;; Return nil response in case of error
         (catch Exception e
-          nil)))))
+          nil))
```

Comment:
Old code, but silently catching exceptions always gets reviewer's attention :)
Is this ok or should there be a log print? check-interface-zip throws an exception with invalid package details.

Reply:
log added